src/components/interaction-designer/clipboard/shared/BlockActionButtons.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="isFocused" class="row blockActionButtonRoot">
+  <div v-if="isFocused" class="block-action-buttons row blockActionButtonRoot">
```

Comment:
Maybe we could reuse the new `block-action-buttons` class that you added for styles as well?
Especially, as blockActionButtonRoot is not kebab-case.

Reply:
:heavy_check_mark: